packages/components/addon/components/hds/table/index.js

```diff
@@ -50,7 +60,7 @@ export default class HdsTableIndexComponent extends Component {
    * @param valign
    * @type {string}
    * @default 'top'
-   * @description Determines the vertical alignment of the table cells; options are all of the values accepted by the CSS "vertical-align" property: "top", "middle", "bottom", "baseline", "sub", and "text-top". If no valign is defined, "top" is used.
+   * @description Determines the vertical alignment of the table cells; options are: "top", "middle". If no valign is defined, "top" is used.
```

Comment:
One consideration. If we define the vertical alignment at table level, consumer may not be able to achieve some complex layouts that _may_ require different alignments for different cells in a row.
<img width="1021" alt="screenshot_2125" src="https://user-images.githubusercontent.com/686239/205911134-afc26644-f60c-4bf6-9c81-713742f647b6.png">
Of course it's an edge case, but still. By moving instead the `valign` property to `Td` level (with the default we want) we allow consumers to achieve a more granular alignment. It may be more verbose for the consumers, but for me it's a more straightforward API.

Reply:
So for this one, Jory and I have already figured out that this shouldn't be a table (no table should have this kind of complexity to it). We are going to offer to pair with that team to help them translate this into a different component/element.

However, if teams do want to still use the table components and achieve complex layouts, they do that now by putting `div` elements inside of the table cell, and using that as their styling hook.